frontends/verific/verific.cc

```diff
@@ -2073,8 +2073,10 @@ struct VerificPass : public Pass {
 	{
 		static bool set_verific_global_flags = true;
 
-		if (check_noverific_env())
+		if (check_noverific_env()) {
+			help();
 			log_cmd_error("This version of Yosys is built without Verific support.\n");
```

Comment:
That's not a command syntax error. I don't think we actually want to print the help message here.

Reply:
Is there no interest in printing out the help message even on non Verific builds so that users can "see what you could have won!!11!!"?

Reply:
Wouldn't that be `help verific`?

Reply:
Fair enough. Reverted in d730bba6d